app/controllers/catalog_controller.rb

```diff
@@ -30,6 +31,10 @@ def remove_from_cart
     change_cart(:remove_item, @equipment_model)
```

Comment:
Yea, we could just call `change_cart(:edit_cart_item, @equipment_model, cart.items[@equipment_model.id] - 1)` and something similar for `add_to_cart`. Thoughts?